### PR TITLE
Zaviermiller kkb 39 event card

### PR DESCRIPTION
**Pull request by zaviermiller**
This PR creates the MVP for the `EventCard` component. I wrote a couple initial tests for the component, but there are still a couple things that could be tested. The `EventCard` component **must** have an `event` prop and can also contain an optional `isAdmin` prop which will be used to show the 'more' dots on the admin pages. There is some weirdness in how the description gets cut off, but I think I can figure out a solution before we ship.

Another thing, I mentioned this in Slack, but I decided to move the `Header` and `Footer` components into `_app.tsx` in order to DRY up the code and make it easier to scaffold pages.

While working on this task, I also started to work on the upcoming events page, but I will finish that in a separate branch. Images below

upcoming events page with eventcard component

### CODE REVIEW

File: src/components/EventCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+            height: 360,
+            borderRadius: 25,
+            // boxShadow: "0 3px 5px -1px rgba(0,0,0,.2),0 5px 8px 0 rgba(0,0,0,.14),0 1px 14px 0 rgba(0,0,0,.12)",
+            transition: ".3s ease-out",
+            position: "relative",
+            cursor: "pointer",
+            minWidth: 300,
+        },
+        thumbnailPlaceholder: {
+            background: `#97BBCB url("/${constants.org.images.defaultCard}") no-repeat center`,
```
Comment (kemalfidan) [inline src/components/EventCard/index.tsx]: Good stuff  👍

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,43 +1,111 @@
...
+                            paddingLeft: "10%",
+                        }}
+                    >
+                        <CoreTypography variant="h1" style={{ color: "white" }}>
+                            Upcoming events
+                        </CoreTypography>
+                        <CoreTypography variant="h3" style={{ fontWeight: "normal", color: "white", marginBottom: 30 }}>
+                            Join us for a workday!
+                        </CoreTypography>
+                        <DDate />
```
Comment (kemalfidan) [inline src/pages/index.tsx]: You can remove the Dummy Date here. I just used that to get the exact format that MUI used for formatting the input date.

File: tst/components/EventCard.test.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { shallow } from "enzyme";
```
Comment (kemalfidan) [inline tst/components/EventCard.test.tsx:1]: Really nice tests! I'll have to reference this for future UI components.

**Review by kemalfidan — APPROVED** [review]

File: src/components/EventCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+            backgroundColor: "rgba(0,0,0,.5)",
+            width: "100%",
+            height: "100%",
+            position: "relative",
+            transition: ".5s",
+        },
+        eventDate: {
+            width: 65,
+            height: 65,
+            backgroundColor: "#87CD9B",
```
Comment (kemalfidan) [inline src/components/EventCard/index.tsx]: Can you replace this with colors from theme?

File: src/components/EventCard/index.tsx (hunk 1 above)
Comment (kemalfidan) [reply src/components/EventCard/index.tsx]: After revisiting this - could you replace this hex color as well? I think the blue in the colors is a placeholder. So can replace that blue and add a dark blue in the colors file. You can also make blue the secondary palette in `theme.ts` and use those colors here.

**Review by kemalfidan — CHANGES_REQUESTED** [review]

**Review by rossketron — APPROVED** [review]
Looks great! I'll definitely be referencing your tests when working on mine

**Comment by zaviermiller** [comment]
@kemalfidan going to work today on fixing those changes, and I also noticed that I use inline styling a lot and I should probably use CSS classes

**Review by kemalfidan — APPROVED** [review]
Looks good! This is a core component of the site so good work :)

**Pull request merged**